src/pages/Product/components/ProductInfo/ProductInfo.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import './ProductInfo.css';
+import ProductImg from "../../../../assets/frstProduct.jpg"
+
+const ProductInfo = () => {
+    const [currentIndex, setCurrentIndex] = useState(0);
+
+    const goToPrevSlide = () => {
+        setCurrentIndex((prevIndex) => (prevIndex - 1 + 2) % 2);
+    };
+
+    const goToNextSlide = () => {
+        setCurrentIndex((prevIndex) => (prevIndex + 1) % 2);
+    };
+
+    const updateSlider = () => {
+        const slider = document.querySelector('.slider');
+        const translateX = -currentIndex * 100 + '%';
+        slider.style.transform = `translateX(${translateX})`;
+    };
+
+    useEffect(() => {
+        updateSlider();
+    }, [currentIndex]);
+
+    const [isDescriptionActive, setIsDescriptionActive] = useState(true);
+
+    return (
+        <div className="product-info">
+                <div className="slider-container">
+                    <div className="slider">
+                        <div className="slide">
+                            <img src={ProductImg} alt="product" />
+                        </div>
+                        <div className="slide">
+                            <img src={ProductImg} alt="product" />
+                        </div>
+                    </div>
+                    <button onClick={goToPrevSlide} className="slider-button prev-button">
```

Comment:
make arrows same as on mock-up - save arrows as svg and import here f.e.

Reply:
rdy